packages/excalidraw/components/App.tsx

```diff
@@ -6035,6 +6036,8 @@ class App extends React.Component<AppProps, AppState> {
     this.maybeCleanupAfterMissingPointerUp(event.nativeEvent);
     this.maybeUnfollowRemoteUser();
 
+    this.lastPointerDownEvent = event; // set the pointer down event and store the inital value
```

Comment:
Can't you use `pointerDownState.originInGrid`, which is already populated and pointer down?